co-log/src/Colog/Message.hs

```diff
@@ -45,5 +50,31 @@ logError :: WithLog env Message m => Text -> m ()
 logError = log Error
 
 -- | Prettifies 'Message' type.
-fmtLogMessage :: Message -> Text
-fmtLogMessage Message{..} = "[" <> show messageSeverity <> "] " <> messageText
+fmtMessage :: Message -> Text
+fmtMessage Message{..} = "[" <> show messageSeverity <> "] " <> messageText
```

Comment:
I was thinking about alignment with extra spaces to make all severities take the same account of symbols:
```
[Debug]   Message
[Info]    Message
[Warning] Message
[Error]   Message

Reply:
@vrom911 That would be nice! I like alignment here. I also left some comment regarding prettier formatting under different issue:

* https://github.com/kowainik/co-log/issues/8